common/dna_data/wc_human_dna.txt

```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
+ 		gene_bs_nose_ridge_width={ "nose_ridge_width_pos" 17 "nose_ridge_width_pos" 35 }
+ 		gene_bs_nose_size={ "nose_size_neg" 77 "nose_size_pos" 38 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_neg" 85 "nose_tip_angle_neg" 7 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_pos" 55 "nose_tip_forward_pos" 44 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 57 "nose_tip_width_pos" 47 }
+ 		face_detail_cheek_def={ "cheek_def_02" 156 "cheek_def_01" 119 }
+ 		face_detail_cheek_fat={ "cheek_fat_02_pos" 90 "cheek_fat_01_pos" 9 }
+ 		face_detail_chin_cleft={ "chin_cleft" 160 "chin_cleft" 1 }
+ 		face_detail_chin_def={ "chin_def" 172 "chin_def" 10 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 53 "eye_lower_lid_def" 91 }
+ 		face_detail_eye_socket={ "eye_socket_01" 142 "eye_socket_02" 195 }
+ 		face_detail_nasolabial={ "nasolabial_01" 98 "nasolabial_01" 6 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_pos" 164 "nose_ridge_def_pos" 45 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 255 "nose_tip_def" 16 }
+ 		face_detail_temple_def={ "temple_def" 255 "temple_def" 221 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_02" 179 "brow_wrinkles_02" 97 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_03" 51 "eye_wrinkles_01" 232 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_02" 41 "forehead_wrinkles_02" 231 }
+ 		expression_other={ "cheek_wrinkles_left_01" 140 "cheek_wrinkles_both_01" 0 }
+ 		complexion={ "complexion_1" 81 "complexion_5" 183 }
+ 		gene_height={ "normal_height" 143 "normal_height" 116 }
+ 		gene_bs_body_type={ "body_fat_head_fat_low" 107 "body_fat_head_fat_full" 144 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 167 "body_shape_pear_half" 0 }
+ 		gene_bs_bust={ "bust_clothes" 133 "bust_shape_1_full" 122 }
+ 		gene_age={ "old_3" 27 "old_2" 166 }
+ 		gene_eyebrows_shape={ "far_spacing_avg_thickness" 182 "far_spacing_low_thickness" 166 }
+ 		gene_eyebrows_fullness={ "layer_2_avg_thickness" 14 "layer_2_avg_thickness" 82 }
+ 		gene_body_hair={ "body_hair_avg" 89 "body_hair_sparse" 168 }
+ 		gene_hair_type={ "hair_straight" 212 "hair_wavy" 120 }
+ 		gene_baldness={ "no_baldness" 0 "no_baldness" 127 }
+ 		eye_accessory={ "normal_eyes" 207 "normal_eyes" 207 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "normal_eyelashes" 79 "normal_eyelashes" 79 }
+ 		gene_bs_ear_lenght={ "floppy_ear_length" 0 "floppy_ear_length" 0 }
+ 		special_eyebrows={ "no_eyebrows" 255 "no_eyebrows" 154 }
+ 		special_eyes={ "no_special_eyes" 0 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_fangs" 0 "mouth_fangs" 0 }
+ 		static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+ 		gene_skin_value={ "skin_value_pos" 47 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 113 "creature_human" 113 }
+        }
+    }
+	enabled=yes
+}
+
+liam_greymane_dna = {
```

Comment:
Liam has hair color closer to red.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123136308-d8dac980-d463-11eb-85f5-cfd23b79ee5f.png)

</details>

Plus, this hairstyle is too bushy.

Reply:
liam is blonde 100%

Reply:
DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192